test/src/single_thread_matrix_calculation_test.cpp

```diff
@@ -3,51 +3,57 @@
 #include <gtest/gtest.h>
 
 #include <cmath>
+#include <iostream>
 
 #include "matrix.h"
 
 namespace mca {
 namespace test {
 class TestSinglThreadCalculation : public testing::Test {
 protected:
-    Matrix<> one, negOne, output, a, b, c, d;
+    Matrix<> one, negOne, output, a, b, c, d, sym, antisym;
```

Comment:
antisym: Could be renamed to skewsym here.

Reply:
antisymmetric is also OK.